modules/services/xidlehook.nix

```
@@ -153,6 +153,7 @@ in {
         ExecStart = "${script}";
       };
       Install.WantedBy = [ "graphical-session.target" ];
+      Restart = "always";
```

Wondering if we should change make this different when `oneshot`

```suggestion
      Restart = if cfg.once then "on-failure" else "always";
```

---

https://github.com/systemd/systemd/pull/13754 since `always` isn't supported, but they can get a restart benefit from `on-failure`

---

Good catch! Perhaps we should keep the existing behaviour for when `cfg.once` is set, though? `Restart=none` is the default. See updated code.

---

That's probably fine, if someone wants to add the on-failure for `oneshot` they can.